### PR TITLE
Unpin pipfile versions

### PR DESCRIPTION
**Pull request by rfoliva**
Changes:
* Include .vscode to the list of editors in .gitignore file.
* Unpin versions of all the packages in Pipfile to help solve conflicts, except for Click and libtmux. Problems were related to Sphinxs and kaptan.
* Cleaned up Pipfile, by removing duplicate entries from packages in dev-packages section.
* Updated requirements files to match stable environment from Pipfile.lock.

"make test" generated some warnings, but passes.

### CODE REVIEW

**Comment by rfoliva** [comment]
Hi @tony.

I think this PR is good for merge and believe that adding Pipfile.lock to keep track of stable environments, while unpinning some versions in Pipfile will make lives much easier.

For stable installation, using locked versions in Pipfile.lock:
`
$ pipenv install --ignore-pipfile --dev
`

For updatred installation, using unlocked versions and using Pipfile settings:
`
$ pipenv install --skip-lock --dev
`

I hope this helps!

File: tmuxp/workspacebuilder.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf8 -*-
+# -*- coding: utf-8 -*-
```
Comment (tony) [inline tmuxp/workspacebuilder.py]: Nice find

**Comment by tony** [comment]
Thank you. I am fairly new to Pipfile version dependencies and locking. 

> For stable installation, using locked versions in Pipfile.lock:
> $ pipenv install --ignore-pipfile --dev
>
> For updatred installation, using unlocked versions and using Pipfile settings:
> $ pipenv install --skip-lock --dev

We may want to update README with this.

File: tmuxp/workspacebuilder.py (hunk 1 above)
Comment (rfoliva) [reply tmuxp/workspacebuilder.py]: That was flake8 and the fact that my pull request reduced the coverage. This file came up as the main offender and this was the first thing flake8 pointed out. I figured I would try and update the pull request and see if that improved coverage results, which it did.

**Comment by rfoliva** [comment]
I will update the documentation in a PR to follow. I agree that if Pipfiles are present, pipenv usage and how to improve development workflow with it should be addressed to help package maintainers and contributors.

Thanks for merging! Glad to be helping.

**Pull request merged**